hw/ip/spi_device/lint/spi_device.waiver

```diff
@@ -123,6 +123,16 @@ waive -rules {NOT_USED NOT_READ} -location {spi_device.sv} \
       -regexp {'sub_(sram|p2s)_.*\[1\]' is not (used|read)} \
       -comment "CmdParse does not have SRAM intf"
 
+#### Intented Terminal States
+waive -rules {TERMINAL_STATE} -location {spi_cmdparse.sv} \
+      -regexp {StWait} \
+      -comment "CmdParse stays StWait until CSb de-assertion is received"
+
+#### Integer use
+waive -rules {INTEGER} -location {spi_cmdparse.sv} \
```

Comment:
i really don't understand why the tool is complaining about this..you're just indexing into an array basically right?

Reply:
yes. It is widely used scheme. Don't know why the lint tool complains it.